src/Tests/Analyzers.Tests/RCS1085UseAutoPropertyTests.cs

```diff
@@ -919,6 +919,35 @@ class C
 class MyAttribute : Attribute
 {
 }
+");
+    }
+
+    [Fact, Trait(Traits.Analyzer, DiagnosticIdentifiers.UseAutoProperty)]
+    public async Task TestFix_WithLeadingWhitespace()
+    {
+        await VerifyDiagnosticAndFixAsync(@"
+class C
+{
+    public int [|P|]
+        {
+            get
```

Comment:
I'm not able to reproduce the bug with this unit test. I'm able to reproduce it only after I add two empty lines before `get`.
```suggestion
        {


            get
```